dataeng/jobs/analytics/UserActivity.groovy

```diff
@@ -0,0 +1,28 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.date_interval_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+
+class UserActivity {
+    public static def job = { dslFactory, extraVars ->
```

Comment:
I mention it below, but I wonder if we can call this something like 'allVars' instead of 'extraVars'.   We use extraVars elsewhere to mean the vars that are in addition to the common vars.    Here it's not clear that they are extra, as they are the only vars.